demo/abell.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   sourcePath: 'src',
   destinationPath: 'dist',
   contentPath: 'content',
-  globalVariables: {
+  globalMeta: {
```

Comment:
`globalMeta` object can now be accessed from any HTML file with `{{ globalMeta.siteName }}`